cql2pgjson/src/test/java/org/folio/cql2pgjson/ForeignKeyGenerationIT.java

```diff
@@ -64,6 +65,20 @@ public void foreignKeySearchParentChild() throws Exception {
     assertThat(cqlb("tablea.name == test1"), containsInAnyOrder("x1"));
   }
 
+  @Test
+  public void foreignKeySearchMulti() throws Exception {
+    assertThat(cql("tablec.cindex == z1"), containsInAnyOrder("test1"));
+  }
+  @Test
+  public void foreignKeySearchMulti2() throws Exception {
+    assertThat(cql("tablec.cindex == z2"), containsInAnyOrder("test2"));
+  }
+
+  @Test
+  public void foreignKeySearchMultiWild() throws Exception {
+    assertThat(cql("tablec.cindex == *"), containsInAnyOrder("test1", "test2"));
+  }
+
```

Comment:
These are tests from parent (table a) to child (table c).
Can you also add at least one test for the other direction?

    cqlc("tablea.name == test1")